build.sbt

```diff
@@ -1384,6 +1385,20 @@ lazy val `jna-wrapper` = project
     }
   )
 
+lazy val `poi-wrapper` = project
+  .in(file("lib/java/poi-wrapper"))
+  .settings(
+    frgaalJavaCompilerSetting,
+    version := "0.1",
+    autoScalaLibrary := false,
+    libraryDependencies ++= Seq(
+      "org.apache.poi" % "poi-ooxml" % poiOoxmlVersion
+    ),
+    assemblyMergeStrategy := { case _ =>
+      MergeStrategy.preferProject
+    }
```

Comment:
- the commit 967e2ce643 starts with following idea
- define `poi-wrapper` fat JAR
- use that `poi-wrapper` in `std-table` instead of direct dependecy on poi

Reply:
- done by @Akirathan at 179aff3b1a001cbc4033c05879c3a9d728661179
- thank you - it's a complicated change - I could never do it myself
- but  the CI is green!
- finally we can move pass this JNI struggle...